client/src/GlobalStyles.js

```diff
@@ -13,7 +13,7 @@ function GlobalStyles() {
         body {
           font-size: 16px;
           margin: 0;
-          background: ${theme.colors.action};
+          background: ${theme.colors.white};
```

Comment:
Please do not name colors like colors. Better is primary, or something more general.